source/browseMode.py

```diff
@@ -1760,6 +1760,19 @@ def script_previousColumn(self, gesture):
 	# Translators: the description for the previous table column script on browseMode documents.
 	script_previousColumn.__doc__ = _("moves to the previous table column")
 
+	def script_toggleIncludeLayoutTables(self,gesture):
+		if config.conf["documentFormatting"]["includeLayoutTables"]:
+			# Translators: The message announced when toggling the include layout tables document formatting setting.
+			state = _("include layout tables off")

```

Comment:
Maybe "layout tables off" or "layout tables disabled" might be more easy to understand.

Reply:
I went the layout tables off route. Also fixed the translator comments in the process.